.github/workflows/ci.yml

```
@@ -2,299 +2,37 @@ name: CI
-        run: cd rust && cargo semver-checks check-release || true
-      - name: Typo check
-        uses: crate-ci/typos@v1
+      - name: Run clippy
+        run: cargo clippy -- -D warnings
+      
 
-  # ─── Rust Coverage ────────────────────────────────────────────────
-  rust-coverage:
-    name: Rust Coverage
+  phenotype-validate:
     runs-on: ubuntu-latest
```

actionlint (1.7.11)

[error] when a reusable workflow is called with "uses", "runs-on" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "phenotype-validate"

(syntax-check)